DependencyInjection/Configuration.php

```diff
@@ -18,8 +18,8 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('lifo_typeahead');
+        $treeBuilder = new TreeBuilder('lifo_typeahead');
+        $rootNode = $treeBuilder->getRootNode();
```

Comment:
If you do that you'll lose backwards compatibility. I haven't gotten around to doing this to typeahead, but I do it in other projects:

```php
        if (Kernel::VERSION_ID < 40000) {
            $treeBuilder = new TreeBuilder();
            $rootNode = $treeBuilder->root('lifo_typeahead');
        } else {
            $treeBuilder = new TreeBuilder('lifo_typeahead');
            $rootNode = $treeBuilder->getRootNode();
        }
```